kolibri/plugins/learn/assets/src/my_downloads/views/DownloadsList/index.vue

```diff
@@ -28,17 +28,18 @@
           <tbody>
             <tr
               v-for="download in Object.values(downloads)"
-              :key="download.node_id"
```

Comment:
You are now keying all selection of the download requests by the `id` rather than what seemed to be intended here, which is the `contentnode_id` - this is not necessarily bad, but will just need some additional double checking that at no point are we trying to do some sort of look up of content nodes by the selected ids directly.